### ISSUE TITLE
Exception while running docker container

### ISSUE THREAD

**Issue by xdx505** [ijo42/GLauncherDockered#1]
Hello! I'm trying to run docker-compose, but I have this exception in running docker container:
```
2022-04-03T03:30:54.148068100Z 06:30:54.147 [INFO ] GravitLauncher (fork sashok724's Launcher) LaunchServer v5.2.9-1 stable (build #1)
2022-04-03T03:30:54.149626700Z 06:30:54.149 [INFO ] License for LaunchServer GPLv3. SourceCode: https://github.com/GravitLauncher/Launcher
2022-04-03T03:30:54.522366500Z Apr 03, 2022 6:30:54 AM org.jline.utils.Log logr
2022-04-03T03:30:54.522403500Z WARNING: Unable to create a system terminal, creating a dumb terminal (enable debug logging for more information)
2022-04-03T03:30:54.676509600Z 06:30:54.676 [INFO ] JLine2 terminal enabled
2022-04-03T03:30:54.678765700Z 06:30:54.678 [INFO ] Creating LaunchServer config
2022-04-03T03:30:54.746319400Z LaunchServer address(default: localhost):
2022-04-03T03:30:54.753777000Z Exception in thread "main" org.jline.reader.EndOfFileException
2022-04-03T03:30:54.754683900Z  at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:653)
2022-04-03T03:30:54.754876900Z  at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:446)
2022-04-03T03:30:54.754887200Z  at pro.gravit.utils.command.JLineCommandHandler.readLine(JLineCommandHandler.java:61)
2022-04-03T03:30:54.754890900Z  at pro.gravit.launchserver.LaunchServerStarter.generateConfigIfNotExists(LaunchServerStarter.java:230)
2022-04-03T03:30:54.754894200Z  at pro.gravit.launchserver.LaunchServerStarter.main(LaunchServerStarter.java:114)
2022-04-03T03:30:54.804752700Z [cmd] s6-setuidgid exited 1
2022-04-03T03:30:54.813857000Z [cont-finish.d] executing container finish scripts...
2022-04-03T03:30:54.815698900Z [cont-finish.d] done.
2022-04-03T03:30:54.817168100Z [s6-finish] waiting for services.
2022-04-03T03:30:55.023448300Z [s6-finish] sending all processes the TERM signal.
2022-04-03T03:30:58.029788400Z [s6-finish] sending all processes the KILL signal and exiting.
```
My docker-compose file:
```
version: "3"

services:
  launchserver:
    image: ijo42/glauncher:latest
    container_name: launchserver
    restart: unless-stopped
    environment:
      - TZ=Europe/Moscow # Timezone
      - PUID=1000 # User ID
      - PGID=1000 # Group ID
    ports:
      - 9274:9274
    volumes:
      - /gravit-launcher:/app/launchserver
```

Path `/gravit-launcher` should be empty or contains any files? In my case, it empties.

**Comment by xdx505** [comment]
When I clone all git project - the issue was fixed. TY

**Comment by ijo42** [comment]
fix docs at 24de2b0f844a0ce8fa32f11d3044ad0815225ae5
